src/components/ToolMenu/FeatureInfo/PaginationToolbar/index.tsx

```diff
@@ -86,6 +121,41 @@ export const PaginationToolbar: React.FC<PaginationToolbarProps> = ({
     copy(JSON.stringify(Object.fromEntries(props)));
   };
 
+  const onEditFeatureBtnClick = () => {
+    if (!layer || !map) {
+      return;
+    }
+    const selectedFeatureClone = selectedFeature.clone();
+    const geojsonFeatureString = new OlFormatGeoJSON().writeFeature(selectedFeatureClone);
+
+    try {
+      const geojsonFeature = JSON.parse(geojsonFeatureString) as Feature;
+      const editLayer = DigitizeUtil.getDigitizeLayer(map);
+      if (editLayer) {
```

Comment:
Absolutely minor, but inverting the conditions (and returning) might increase readability.